packages/schemas/src/date/types.ts

```diff
@@ -9,4 +9,5 @@ export interface DateSchema extends Schema {
   characterSpacing: number;
   fontColor: string;
   backgroundColor: string;
+  locale?: string;
```

Comment:
Sorry, I ended up reverting `localeForUI` back to `locale`.

I realized that when the format is something like `MM/dd/yyyy hh:mm aa`, `localeForUI` is no longer just a property for UI since it is also used to determine the `aa` part of the value.